neurom/apps/morph_stats.py

```diff
@@ -186,14 +188,14 @@ def _fill_stats_dict(data, stat_name, stat):
         for mode in modes:
             stat_name = _stat_name(feature_name, mode)
             stat = eval_stats(feature, mode)
-            _fill_stats_dict(stats[neurite_type.name], stat_name, stat)
+            _fill_stats_dict(stats[neurite_type.name], stat_name, stat, feature_name)
 
     for feature_name, modes in config.get('neuron', {}).items():
         feature = nm.get(feature_name, neurons)
```

Comment:
Thinking about it, we could define another function `get_feature_and_shape`, similar to `get`, that returns the feature and the shape? (it could be private if we don't want to expose it)
In this way we can pass the shape to `_fill_stats_dict`, and we don't need to call the private function `_find_feature_func` to get the function and the shape after we have already got the feature value.

Reply:
I have done what you have suggested. I haven't made the function public since we don't need to and so that we can still change the implementation in the future.